scripts/src/admin/services.ts

```diff
@@ -693,18 +693,23 @@ export async function changeOffer(body: ChangeOfferData, params: { offer_id: str
 	await offer.save();
 }
 
-type UpdateAppConfigRequest = Request & {
-	body: ApplicationConfig
-};
+type AppConfig = ApplicationConfig & { [limits: string]: number };
+type UpdateAppConfigRequest = Request & { body: AppConfig; };
 
 export const updateAppConfig = async function(req: UpdateAppConfigRequest, res: Response) {
+	const config: AppConfig = req.body;
+	const isLimitsNumbers = (configObj: AppConfig, limitName: string) => typeof configObj.limits[limitName as keyof AppConfig["limits"]] === "number";
+	if (!config.limits || !Object.keys(config.limits).every(isLimitsNumbers.bind(null, config))) {
```

Comment:
Personally, I'd put `isLimitsNumbers.bind(null, config)` into a const named something like `isConfigLimitsValid` which makes this a little bit more readable and descriptive (and in the future will hold other tests we might want to do on these user inputs. But that's my _personal_ preference.